x/ecosystem-incentive/keeper/reward_service.go

```diff
@@ -77,7 +77,7 @@ func (k Keeper) SetRewardStore(ctx sdk.Context, rewardStore types.RewardStore) e
 
 	store := ctx.KVStore(k.storeKey)
 	prefixStore := prefix.NewStore(store, []byte(types.KeyPrefixRewardStore))
-	prefixStore.Set(rewardStore.SubjectAddr.AccAddress(), bz)
+	prefixStore.Set(rewardStore.SubjectAddr.AccAddress().Bytes(), bz)
```

Comment:
Is there any specific reason doing bytes conversion here?

Reply:
yes, i did it for the purpose of the obviousness since I confused how i treated StringAccAddress and AccAddress this time.

Reply:
Okay